packages/i18n/src/locales/en.i18n.json

```diff
@@ -3080,7 +3080,7 @@
   "LDAP_Sync_User_Data_Channels_BaseDN": "LDAP Group BaseDN",
   "LDAP_Sync_User_Data_Channels_Description": "Enable this feature to automatically add users to a channel based on their LDAP group. If you would like to also remove users from a channel, see the option below about auto removing users.",
   "LDAP_Sync_User_Data_Channels_Enforce_AutoChannels": "Auto Remove Users from Channels",
-  "LDAP_Sync_User_Data_Channels_Enforce_AutoChannels_Description": "**Attention**: Enabling this will remove any users in a channel that do not have the corresponding LDAP group! Only enable this if you know what you're doing.",
+  "LDAP_Sync_User_Data_Channels_Enforce_AutoChannels_Description": "**Attention**: Enabling this will remove any users in a channel that do not have the corresponding LDAP group! Only enable this if you know what you'r doing.",
```

Comment:
```suggestion
  "LDAP_Sync_User_Data_Channels_Enforce_AutoChannels_Description": "**Attention**: Enabling this will remove any users in a channel that do not have the corresponding LDAP group! Only enable this if you know what you're doing.",
```